src/Console/Test/console_test.cpp

```diff
@@ -54,8 +54,14 @@ TCL_TEST(console_proc) {
 
 TCL_TEST(console_multiline) {
   FOEDAG::TclConsoleWidget *console = FOEDAG::InitConsole(interp);
-  QString res = console->getPrompt() + "proc test {} {\nputs test\n}\n" +
-                "test\n" + console->getPrompt();
+  QString script =
+      R"("
+          proc test {} {
```

Comment:
current implementation send every character. So it will not work since you put many spaces.